mujoco_ros_sensors/CMakeLists.txt

```diff
@@ -4,16 +4,13 @@ project(mujoco_ros_sensors)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-
-# Disable optimization to prevent false-positives with memory analisys tools (e.g. asan or valgrind)
-if (ENABLE_SANITIZER OR NO_OPTIM)
-  set(CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} -O0 -fno-inline")
-  set(CMAKE_BUILD_TYPE "RelWithDebInfo")
+if(NOT CMAKE_BUILD_TYPE)
+  message(STATUS "CMAKE_BUILD_TYPE not set. Defaulting to 'Release'.")
+  set(CMAKE_BUILD_TYPE "Release")
 endif()
 
 if (ENABLE_SANITIZER)
```

Comment:
The mentioned commit also introduces the Sanitizer build type for the sensors and the control packages' CMakeLists